src/cordova/apple/xcode/ios/Outline/Outline-Info.plist

```diff
@@ -2,10 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
```

Comment:
Same comment as for the config.XML. Perhaps leave a placeholder so we can build without injection.
And to make it clear that it exists, but it doesn't have the actual value.